pyproject.toml

```diff
@@ -131,21 +139,6 @@ build = "mkdocs build --clean --strict" # hatch run docs:build
 # Start the development server for the documentation with `mkdocs`:
 serve = "mkdocs serve" # hatch run docs:serve
 
-[tool.hatch.envs.tests]
-# Dependencies to be installed in the `tests` virtual environment.
-dependencies = [
-    "pytest==8.3.2",        # to run the tests
-    "coverage==7.6.1",      # to generate coverage reports
-    "time-machine==2.15.0", # to select an arbitrary date and time for testing
-    "pypdf==4.3.1",         # to read PDF files
-]
-path = ".venv-tests"
-
-[tool.hatch.envs.tests.scripts]
-# Run the tests:
-run = "pytest" # hatch run tests:run  
-# Run the tests and generate the coverage report as HTML:
-run-and-report = "coverage run -m pytest && coverage report && coverage html --show-contexts" # hatch run tests:coverage
```

Comment:
Is there a particular reason for removing `.venv-tests`?

Reply:
Yes, I moved test dependencies it to `.venv`. Keeping two virtual environments for development and tests was very hard to use; you had to switch between them all the time. Usually, testing and development take place at the same time. That's why I did it.